src/vm_pages.rs

```diff
@@ -304,12 +304,6 @@ enum VmRegionType {
     Imsic,
```

Comment:
something to consider, not necessarily for this PR, is to adapt the whole IMSIC bind/unbind/rebind bits to internally use the new block/remove flow. that would allow us to completely get rid of the existing "unassignment" flow.